.releaserc

```diff
@@ -21,7 +27,8 @@
               "dist/**/*.{js,css}",
               "build/**/*.{js,css}"
             ]
-          }
+          },
+          {"path": "docs/CHANGELOG.md", "label": "Changelog"}
```

Comment:
@davidornelas11 We were going to include the CHANGELOG.md in the root of the package instead of in the docs/ folder. Does that need to be  updated?